src/sentry/static/sentry/less/group-detail.less

```diff
@@ -711,6 +711,12 @@
     color: @gray-dark;
   }
 
+  .icon-alert {
```

Comment:
This isn't a very specific selector, but from what I can tell there are a lot of places where we keep the selectors pretty generic. If my impression here is wrong, I'm happy to change it to something more specific (e.g. `.latency.icon-alert`) or something.